modules/dxp/apps/app-builder-workflow/app-builder-workflow-web/test/pages/apps/edit/EditApp.es.js

```diff
@@ -297,188 +289,296 @@ describe('EditApp', () => {
 				</AppContextProviderWrapper>
 			);
 
-			await waitForElementToBeRemoved(() =>
-				document.querySelector('span.loading-animation')
-			);
+			const nameInput = result.getByPlaceholderText('untitled-app');
 
-			await fireEvent.click(getAllByText('en-US')[1]);
+			result.getByText('new-workflow-powered-app');
```

Comment:
Is this doing something?

Reply:
The use of `get*()` doesn't need to call `expect()`. I just use this to check if the element exist, if it doesn't exist the operation will fail. But just when I use the `find*()` the operation will not fail. Makes sense? @inacionery @gabrielpramos

Reply:
If it isn't clear for you, I can use the `expect()` calling with `toBeInTheDocument()` or  `toBeTruthy()` validations, the behaviour will be the same.

Reply:
alright, we can keep like this. thanks!